presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeWindows.java

```diff
@@ -393,32 +475,50 @@ public void testNotMergeDifferentNullOrdering()
                 "SUM(discount) OVER (PARTITION BY suppkey ORDER BY orderkey ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) sum_discount_A " +
                 "FROM lineitem";
 
-        Window windowC = new Window(
-                ImmutableList.of(new SymbolReference("suppkey")),
-                ImmutableList.of(new SortItem(new SymbolReference("orderkey"), SortItem.Ordering.ASCENDING, SortItem.NullOrdering.FIRST)),
-                Optional.of(commonFrame));
+        PlanMatchPattern tableScan = tableScan(
```

Comment:
It seems like this tableScan is common to a lot of tests, maybe you could extract it?